### PR TITLE
feat(Authorization): Added support to read keystore from central configuration

### PR DESCRIPTION
**Pull request by nutanv1contr**
1. Added support to read jwt-keystore.jks file from etc/sw360 folder, if not found then it will search from default project resource path. 
2. Read keystore secret key from application.yml . 

closes: #691

### CODE REVIEW

File: rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360AuthorizationServerConfiguration.java [hunk 1]
```diff
@@ -95,8 +105,12 @@ public TokenStore tokenStore() {
 
     @Bean
     protected JwtAccessTokenConverter jwtAccessTokenConverter() {
+        Resource resource = new FileSystemResource(new File("/etc/sw360/jwt-keystore.jks"));
```
Comment (smrutis1) [inline rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360AuthorizationServerConfiguration.java]: Please remove the hard-coded file path and read it from `CommonUtils.java`

Comment (nutanv1contr) [reply rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360AuthorizationServerConfiguration.java]: Thanks @smrutis1 , I have incorporated the changes.

**Comment by mcjaeger** [comment]
note that build runs fine, but does not exit cleanly, should be OK, restarted anyways.

**Comment by smrutis1** [comment]
Tested the PR. 
1) Reading the keystore password from `application.yml`.
1) Able to read the keystore from central configuration path(`/etc/sw360`).

**Pull request merged**